1-org/envs/shared/terraform.example.tfvars

```diff
@@ -30,6 +30,8 @@ default_region = "us-central1"
 
 scc_notification_name = "scc-notify"
 
+hub_and_spoke = true
```

Comment:
shouldn't it be `enable_hub_and_spoke` ?

Reply:
Make sence, I will update var name